src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -79,17 +79,28 @@ object EMD extends DebugEnhancedLogging {
   }
 
   private def addPrivacySensitiveRemark(emd: EasyMetadata, agreementsXml: Elem): Unit = {
-    val signerId = agreementsXml \ "depositAgreement" \ "signerId"
-    val easyAccount = (signerId \@ "easy-account").toOption
-    val fullname = signerId.text
+
+    def userNamePart: String = {
+      val signerId = agreementsXml \ "depositAgreement" \ "signerId"
+      (signerId.text, (signerId \@ "easy-account").toOption, (signerId \@ "email").toOption) match {
```

Comment:
Please do an 'extract value' on the account and email and add `.text` to the `val signerId` for better readability

Reply:
Also add a `toOption` to the `signerId`, as with the account and email. That way the pattern match is easier, I think